src/components.jl

```diff
@@ -35,7 +35,7 @@ function add_component!(
     components::Components,
     component::T;
     skip_validation = false,
-    deserialization_in_progress = false,
+    allow_existing_time_series = false,
```

Comment:
@daniel-thom This broke the API... we needed to add a deprecation

Reply:
As far as I know, PowerSystems is the only module that relies on this, so I think we're safe changing it. It does not affect `PowerSystems.add_components`.